src/components/Navbar.js

```diff
@@ -3,8 +3,8 @@ import { Link } from 'react-router-dom';
 
 function Nav() {
   return (
-    <div>
-      <h1>BOOKSTORE CMS</h1>
+    <div className="nav-container">
+      <div><h1>BOOKSTORE CMS</h1></div>
```

Comment:
- [ ] please make sure the navbar looks EXACTLY, the same as the example provided by Microverse, as you can see in the below example the user icon is different in the example.
your project:
![afbeelding](https://user-images.githubusercontent.com/39240820/232134898-0c7f5b2a-88be-44bc-84a8-5dc9ff054dad.png)
Zeplin example:
![afbeelding](https://user-images.githubusercontent.com/39240820/232134979-26e63f62-c761-4491-9bcb-34c143ce4be5.png)

Reply:
- [ ] As previous reviewer mentioned your design should match the original design. Kindly remove extra spaces and make your bookstore container full screen.

<img width="1436" alt="Screenshot 2023-04-15 at 01 29 43" src="https://user-images.githubusercontent.com/69011963/232148883-22d22c73-4405-4e3f-b8fc-c052f7743a15.png">